pkg/codegen/jenny_crd_yaml.go

```diff
@@ -0,0 +1,213 @@
+package codegen
```

Comment:
We've solved most of this in the app SDK repo – I assume this code is for demonstration only?

Reply:
basically yes, i would love nothing more than unifying the sdks we're making

i can't recall exactly what it was that led me to copy these types over (or maybe it was just [this](https://github.com/grafana/grafana/pull/60207#discussion_r1047134398)), but this (and a fair bit of what's currently in `pkg/kindsys/k8ssys`) should be deduplicated.

need to grant the commit bit on grafana for you folks in @grafana/apps-platform-core - cc @daniellee

Reply:
"grant the commit bit on grafana" ✅

Reply:
Thank you!